docs/how-to-guides/index.rst

```diff
@@ -16,6 +16,7 @@ aimed at active users of Renku and target specific use-cases or common issues.
    migrating-projects-to-renku
    collaboration
    git-tips
+   s3
```

Comment:
sorry, just noticed this - I think it should go into the toc in `sessions.rst`

Reply:
No problem at all. That makes perfect sense.